producer/pdu_session.go

```diff
@@ -392,6 +392,15 @@ func HandlePDUSessionSMContextUpdate(smContextRef string, body models.UpdateSmCo
 			logger.PduSessLog.Infoln("The SMContext State should be Active State")
 			logger.PduSessLog.Infoln("SMContext state: ", smContext.SMContextState.String())
 		}
+		// If the PDU session has been released, skip sending PFCP Session Modification Request
+		if smContext.SMContextState == smf_context.InActivePending {
```

Comment:
it's not necessary, `HandlePDUSessionSMContextUpdate()` will handle this case later.

Reply:
Well, but I think it would be easier to read the code if you indicate that you want to http rsp here.